app.json.example

```diff
@@ -3,7 +3,6 @@
     "name": "WordyWorld",
     "slug": "wordy-world",
     "privacy": "unlisted",
-    "sdkVersion": "36.0.0",
```

Comment:
It disappeared from the diff in `app.json` when upgrading from SDK 38 to 39 with `expo upgrade`

```
✔ Removed deprecated sdkVersion field from app.json.
```